x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -437,6 +476,18 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
               </EuiFormRow>
             </Fragment>
           )}
+          <EuiFormRow
+            label={i18n.translate('xpack.ml.dataframe.analytics.create.modelMemoryLimitLabel', {
+              defaultMessage: 'Model memory limit',
+            })}
+          >
+            <EuiFieldText
```

Comment:
Could the default value (or the one from the endpoint) be used as a placeholder?

![image](https://user-images.githubusercontent.com/7405507/68949070-29cb0c00-07b1-11ea-81f0-ab4e2d24006e.png)

Reply:
Placeholder added: f8c50cfe758da1782208983286bf40e4fc06af1f